hoodie-client/src/main/java/com/uber/hoodie/config/HoodieCompactionConfig.java

```diff
@@ -45,6 +45,7 @@
   public static final String CLEANER_COMMITS_RETAINED_PROP = "hoodie.cleaner.commits.retained";
   public static final String MAX_COMMITS_TO_KEEP = "hoodie.keep.max.commits";
   public static final String MIN_COMMITS_TO_KEEP = "hoodie.keep.min.commits";
+  public static final String COMMITS_ARCHIVAL_BATCH_SIZE_PROP = "hoodie.commits.archival.batch";
```

Comment:
Move archive configs to a new Config class HoodieArchiveConfig ?

Reply:
Don't want to move to a new config for 1 addition..

Reply:
I am only suggesting this because this is yet another tech debt which is being piled-up instead of getting fixed.  This new config along with the 2 configs above this config are for archiving.

Reply:
Yeah, I agree with you on not having more tech debt. What I'm saying is slightly different - Don't think 1 config + remaining 2 configs warrant for a class separation. WDYT ?

Reply:
I was thinking  its ok to have smaller config classes. Apart from archiving, there is Cleaner related config too which can be moved away. But if you feel it is too fine-grained, may be change the class name from HoodieCompactionConfig to something more generic for now.

Reply:
Changing the config name is backwards incompatible change. Let me open another ticket to revisit and clean up configs.

Reply:
https://issues.apache.org/jira/browse/HUDI-97

Reply:
Sounds good.